app/models/spam_report.rb

```diff
@@ -0,0 +1,78 @@
+class SpamReport
+
+  attr_reader :recent_date, :new_date
+
+  def self.run
+    self.new.run
+  end
+
+  def initialize
+    @recent_date = 14.days.ago
+    @new_date = 1.day.ago
+  end
+
+  def run
+    spam = {}
+    users.each do |user|
+      new_works, score = score_user(user)
+      if score > ArchiveConfig.SPAM_THRESHOLD
+        spam[user.id] = { score: score, work_ids: new_works }
+      end
+    end
+    spam = Hash[spam.sort_by { |user_id, info| info[:score] }.reverse]
```

Comment:
Unused block argument - `user_id`. If it's necessary, use `_` or `_user_id` as an argument name to indicate that it won't be used.